docs/spec.md

```diff
@@ -3703,30 +3704,38 @@ More formally, `result[i0, ..., iR-1]` is equal to:
 
 #### Inputs
 
-| Label | Name                | Type                                         | Constraints      |
-|-------|---------------------|----------------------------------------------|------------------|
-| (I1)  | `operand`           | tensor                                       | (C1), (C2), (C4) |
-| (I2)  | `padding_value`     | 0-dimensional tensor                         | (C1)             |
-| (I3)  | `edge_padding_low`  | 1-dimensional tensor constant of type `si64` | (C2), (C4)       |
-| (I4)  | `edge_padding_high` | 1-dimensional tensor constant of type `si64` | (C2), (C4)       |
-| (I5)  | `interior_padding`  | 1-dimensional tensor constant of type `si64` | (C2-C4)          |
+| Label | Name                | Type                                         | Constraints   |
+|-------|---------------------|----------------------------------------------|---------------|
+| (I1)  | `operand`           | tensor or quantized tensor                   | (C1), (C3-C7) |
+| (I2)  | `padding_value`     | 0-dimensional tensor or quantized tensor     | (C4-C6)       |
+| (I3)  | `edge_padding_low`  | 1-dimensional tensor constant of type `si64` | (C1), (C3)    |
+| (I4)  | `edge_padding_high` | 1-dimensional tensor constant of type `si64` | (C1), (C3)    |
+| (I5)  | `interior_padding`  | 1-dimensional tensor constant of type `si64` | (C1-C3)       |
 
 #### Outputs
 
-| Name     | Type   | Constraints |
-|----------|--------|-------------|
-| `result` | tensor | (C1)        |
+| Name     | Type                       | Constraints |
+|----------|----------------------------|-------------|
+| `result` | tensor or quantized tensor | (C3-C6)     |
 
 #### Constraints
 
 <!-- markdownlint-disable line-length -->
-* (C1) `operand`, `padding_value`, `result` have the same element type.
```

Comment:
Similarly, we can keep C1 as is because it applies to both cases is we allow element_type to produce both regular and quantized element types. Then below, we'll only have one case to take care of - "if the operation uses quantized tensors". (The same applies for all other ops below).

Reply:
Let us discuss this [here](https://github.com/openxla/stablehlo/pull/1566#discussion_r1220364289).

Reply:
Moved the discussion thread here: https://github.com/openxla/stablehlo/pull/1567#discussion_r1217167342.